tools/ci/deps-ubuntu.sh

```
@@ -45,19 +45,19 @@ sudo apt-get -qq install -y \
-  sudo apt-get install -y -V ./apache-arrow-apt-source-latest-$(lsb_release --codename --short).deb
-  sudo apt update
-  # Pin to Arrow 22.x to avoid CMake configuration issues with Arrow 23.0.0
-  sudo apt-get install -y --no-install-recommends \
-        libarrow-dev=22.0.0-1 \
-        libparquet-dev=22.0.0-1 \
-        libarrow2200=22.0.0-1 \
-        libparquet2200=22.0.0-1 \
+# Install Apache Arrow
+sudo apt update
+sudo apt-get install -y -V ca-certificates lsb-release wget
+wget https://packages.apache.org/artifactory/arrow/$(lsb_release --id --short | tr 'A-Z' 'a-z')/apache-arrow-apt-source-latest-$(lsb_release --codename --short).deb
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)